### PR TITLE
Setup cypress

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Install cypress on canonical.com
- Setup a single test to access the homepage and accept the cookie policy

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4588

### CODE REVIEW

File: tests/cypress/integration/sample_spec.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+describe("Canonical home page", () => {
+  it("should go to the Canonical homepage and accept the cookie policy", () => {
+    cy.visit("/");
+    cy.acceptCookiePolicy();
```
Comment (bethcollins92) [inline tests/cypress/integration/sample_spec.js:4]: You could add an assertion in here (using the word should) to make it a real test as at the moment you're asking cypress to follow instructions but not checking what you expect to happen. e.g.

`cy.findByText(/Deliver, maintain,secure and sustain/).should("be.visible");` 

as after the modal closes you expect to be able to see this text. (Not a blocker just a suggestion as I know this is only for the set up).

**Review by bethcollins92 — APPROVED** [review]
Great work @petesfrench 🥳 looking good

File: tests/cypress/integration/sample_spec.js (hunk 1 above)
Comment (petesfrench) [reply tests/cypress/integration/sample_spec.js:4]: Cheers Beth! Would it makes sense to also check if the cookie-policy is visible? As we want to close it in this test.

Comment (bethcollins92) [reply tests/cypress/integration/sample_spec.js:4]: I suppose if you're able to click the "Accept blah blah" button then you know it must be there, this test could just be checking it closes as expected when you click that button? although I know it's only to check cypress is actually up and running and isn't a real test

Comment (petesfrench) [reply tests/cypress/integration/sample_spec.js:4]: Makes sense, I will stick with your suggestion for now

**Pull request merged**